This creates an (ultimately empty) pkgincludedir, which throws off
the symlink created in install-data-local.

--- origsrc/libfm-1.1.1/src/Makefile.am	2013-07-22 08:18:58.000000000 -0500
+++ src/libfm-1.1.1/src/Makefile.am	2013-07-22 19:36:57.804286800 -0500
@@ -211,11 +211,6 @@ libfmgtkinclude_HEADERS = \
 	gtk/fm-gtk-marshal.h \
 	$(NULL)
 
-pkginclude_HEADERS = \
-	$(LIBFM_INCLUDES) \
-	$(LIBFM_GTK_INCLUDES) \
-	$(NULL)
-
 EXTRA_LTLIBRARIES = libfm-gtk.la libfm-gtk3.la
 
 lib_LTLIBRARIES = libfm.la @LIBFM_GTK_LTLIBRARIES@
